src/include/kernel/ext_manifest.h

```diff
@@ -5,6 +5,12 @@
  * Author: Karol Trzcinski <karolx.trzcinski@linux.intel.com>
  */
 
+/*
+ * Parts of this file are copied to ext_manifest.h in Linux kernel tree
+ * and to the rimage tree. This duplication is OK because it is part of
+ * a versioned ABI.
+ */
+
 /*
```

Comment:
I wonder if ext_manifest.h and files containing ABI shouldn't be exactly the same copy in all the repos. Change looks good for now.

Reply:
I considered a couple other ways to organize these definitions and making it easier to compare content and propagate changes across repos was definitely one of my concerns. However for now this PR is focused on making the smallest and easiest to review changes to unblock the Zephyr uprev in https://github.com/zephyrproject-rtos/zephyr/pull/37250 and validation in general. If everything goes well this will be the first time that an unmodified SOF commit builds Zephyr CI! The more automation we have, the easier it  should be to re-organize these headers later.